src/chat/KeywordResponses.hpp

```diff
@@ -0,0 +1,38 @@
+// chris busch
+
+#pragma once
+
+namespace chat {
+
+/**
+ * A response message with an assigned weight that's used when
+ * determining a random response to a keyword from the user.
+ */
+struct WeightedResponse {
+    int weight_;
+    char *message_;
```

Comment:
can this be `const`? (if not a std::string)

Reply:
Not right now, but probably once the way they're stored by KeywordResponses they can be